src/nvim/ui.c

```diff
@@ -338,6 +359,18 @@ void ui_line(ScreenGrid *grid, int row, int startcol, int endcol, int clearcol,
   }
 }
 
+void ui_composed_call_raw_line(Integer draw_grid, Integer row,
```

Comment:
isn't "composite" the more common phrasing rather than "composed"?

Reply:
Maybe, but "composed" is a passive/past tense as in "the UI is (being) composed" or "after the line was composed"